specification/security/resource-manager/readme.md

```diff
@@ -153,13 +153,14 @@ input-file:
 - Microsoft.Security/preview/2017-08-01-preview/workspaceSettings.json
 - Microsoft.Security/preview/2019-01-01-preview/regulatoryCompliance.json
 - Microsoft.Security/preview/2019-01-01-preview/serverVulnerabilityAssessments.json
-- Microsoft.Security/preview/2019-01-01-preview/subAssessments.json
 - Microsoft.Security/preview/2019-01-01-preview/automations.json
+- Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```

Comment:
@mikreite Iv'e checked and adding the - Microsoft.Security/preview/2019-01-01-preview/automations.json
fails lintdiff

Reply:
it works. Probably the pushes to the branches re triggered a build, so the issue was transient